src/Microsoft.DocAsCode.Dfm.VscPreview/App.config

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
```

Comment:
for xml file extension, the indent/space size should be 2? you can config in VS.